README.md

```diff
@@ -61,6 +61,8 @@ it takes a promise that resolves to an array, and returns a promise that
 resolves to the expected shape by `connectionType`.
  - `cursorForObjectInConnection` is a helper method that takes an array and a
 member object, and returns a cursor for use in the mutation payload.
+- `offsetToCursor` takes the index of a member object in an array and
+returns a cursor for use in the mutation payload.
```

Comment:
I think maybe these docs should mention that cursors are _supposed_ to be opaque, even though we're exposing some low-level primitives for working with them here. It could be as simple as changing "returns a cursor" to "returns an opaque cursor".